pallets/capacity/src/tests/mock.rs

```diff
@@ -166,12 +171,22 @@ impl pallet_capacity::Config for Test {
 	type UnstakingThawPeriod = ConstU16<2>;
 	type MaxEpochLength = ConstU64<100>;
 	type EpochNumber = u32;
+	type TokenPerCapacity = TokenPerCapacity;
 }
 
 pub fn new_test_ext() -> sp_io::TestExternalities {
 	let mut t = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap();
 	pallet_balances::GenesisConfig::<Test> {
-		balances: vec![(100, 10), (200, 20), (300, 30), (400, 40), (500, 50), (600, 60)],
+		balances: vec![
```

Comment:
In the mocks I set the Token:Capacity ratio to 10 (not 100) which requires these balances be changed.  This also uncovered the bugs.